scripts/updateLicenses.sh

```diff
@@ -3,4 +3,4 @@
 set -e
 set -x
 
-python scripts/updateLicense.py $(go list -json $(glide nv) | jq -r '.Dir + "/" + (.GoFiles | .[])')
+python "$(dirname $0)"/updateLicense.py $(go list -json $(glide nv) | jq -r '.Dir + "/" + (.GoFiles | .[])')
```

Comment:
I mean, I could separate it but this is a trivial enough change that it doesn't really matter. The `updateLicenses` call in gen.go won't work without this change.